assets/src/util/outfront.tsx

```diff
@@ -44,7 +24,7 @@ export const getRotationIndex = (): RotationIndex | null => {
 /**
  * Gets Outfront's unique name/ID for the media player we're running on.
  *
- * For DUPs, this is just a single ID. For triptychs, each of the 3 panes has its own player name.
+ * For DUPs, this is just a single ID.
```

Comment:
Can probably remove this "paragraph" entirely as it was only meaningful in the context of the comparison between DUPs and Triptychs.